### DISCUSSION TITLE
Ideas to Add

### DISCUSSION THREAD

**Discussion by bluecursor** [jacklandrin/OnlySwitch#88] [Ideas]
Switches to Add:

- Do not Disturb
- Change Screen Resolution
- Clean Screen (Black out Screen)
- Display Sleep
- Lock Keyboard
- Clean Clipboard (remove all formatting/make plain text)
- Default Browser
- Smooth Scroll Mouse (Scroll Wheel)

In Back Noises:
- Preview Sounds
- Ability to Add your own Sounds

In Evolution:
- Add Evolution Gallery Section

**Comment by jacklandrin** [comment]
@bluecursor  Thank you for your suggestion. Regarding these, I added comments behind each item.
Switches to Add:

Do not Disturb **(already in shortcut gallery)**
Change Screen Resolution **(no plan so far)**
Clean Screen (Black out Screen) **(please check screen test switch)**
Display Sleep **(I think hotkey comtrol + command + Q is enough easy)**
Lock Keyboard **(also screen test switch)**
Clean Clipboard (remove all formatting/make plain text) **(Copy and paste-without-formatting on a Mac: Cmd-C, Cmd-Opt-Shift-V)**
Default Browser **(no plan so far)**
Smooth Scroll Mouse (Scroll Wheel) **(please check Toggle Scroll Direction in shortcuts gallery, above Ventura Invert Scroll Direction (Ventura))**
In Back Noises:

Preview Sounds **(in plan)**
Ability to Add your own Sounds **(no plan so far)**
In Evolution:

Add Evolution Gallery Section **(in plan)**

**Comment by bluecursor** [comment]
Running Screen Test crashes/quits OnlySwitch. No Error Report displayed. OSX Ventura 13.4.1

Inverting/Toggle Scroll Direction is not what i meant. I was referring to the speed in which a window scrolls.

**Comment by jacklandrin** [comment]
@bluecursor 
I just updated 13.4.1 to test the crash you mentioned, but it couldn't reproduce on my side. May I ask if it crashes every time?
Regarding Smooth Scroll Mouse, I will check if it can be implemented by shortcuts.

**Comment by bluecursor** [comment]
Every time…..

BlueCursor
www.imacommodork.com
Sent with OSX/iOS Spark!

**Comment by jacklandrin** [comment]
emmm, strange
If you could debug in Xcode, and perhaps pull my source code, it would be easier to find out some clues.
